src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -171,6 +172,12 @@ Nethermind is an Ethereum implementation created with the C# .NET tech stack, li
 
 Nethermind also has [detailed documentation](https://docs.nethermind.io), strong dev support, an online community and 24/7 support available for premium users.
 
+### Ethereumjs {#ethereumjs}
+
+It is written in TypeScript and composed of a number of packages, including core Ethereum primitives represented by the the Block, Transaction, and Merkle-Patricia Trie classes and core client components including an implementation of the Ethereum Virtual Machine (EVM), a blockchain class, and the DevP2P networking stack.
+
+Learn more about ethereumjs by reading its [documentation](https://ethereumjs.readthedocs.io/en/latest/)
```

Comment:
We initially had a more cautious text here as well, emphasizing the experimental/research focus of the client. I am not sure where this got lost but we should really stick with that, otherwise we are overpromising here as well.

Reply:
Updated.